README.md

```diff
@@ -239,6 +239,9 @@ The `template` parameter is the name of the AWX job template.
 The `extraVars` parameter is optional, and if specified it is used to
 pass additional variables to the playbook, like with the `--extra-vars`
 option of the `ansible-playbook` command.
+Regardless to the `extraVars` setting, the alert that triggered the AWX 
```

Comment:
I think this should be a separate paragraph. So can you add a leading blank line and end the paragraph with a dot?